src/components/common/AnimationControls.js

```diff
@@ -115,6 +145,38 @@ const AnimationControls = () => {
         </Tooltip>
       )}
 
+      <Tooltip title={t('Move Back')} placement="top">
+        <span>
+          <IconButton
+            onClick={onClickRewind}
+            disabled={canvasIncomplete || !isPaused || intervalCount === 0}
+          >
+            <FastRewindIcon
+              className={`${classes.button} ${
+                !canvasIncomplete && isPaused && intervalCount !== 0
+                  ? classes.rewindButton
+                  : ''
+              }`}
```

Comment:
Try `clsx`, it would simplify your classname to 
```
clsx(classes.button, {classes.rewindButton: !canvasIncomplete && isPaused && intervalCount !== 0})
```